src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+2.2.17
```

Comment:
Does adding a new command call for a minor bump instead of a patch bump? i.e. should this be `2.3.0`?

Reply:
We've never required that, no.